docs/rules/space-infix-ops.md

```diff
@@ -22,21 +25,30 @@ This rule is aimed at ensuring there are spaces around infix operators.
 
 ## Options
 
-This rule accepts a single options argument with the following defaults:
+* `all` set to `always` requires spaces around all operators that do not override this (the default).
+* `all` set to `never` disallows spaces around all operators that do not override this.
+* `all` set to `ignore` doesn't enforce any particular use of spaces around operators that do not override this.
+* The name of every operator (except `in` and `instanceof`)
+  can be used as an option with the values
+  `always` (require spaces), `never` (disallow spaces), or `ignore` (don't care).
+  These options override the `all` setting for that operator.
+  Options for `in` and `instanceof` are not supported because
+  it is not valid to omit spaces around those operators.
+* `int32Hint` set to `true` allows writing `a|0` without spaces (Default `false`).
+  This option is deprecated in favor of specifying `"|": "never"` or `"|": "ignore"`.
+
+For example,
 
 ```json
-"space-infix-ops": ["error", {"int32Hint": false}]
+"space-infix-ops": ["error", {"**": "never", "*": "ignore"}]
 ```
 
-### `int32Hint`
-
-Set the `int32Hint` option to `true` (default is `false`) to allow write `a|0` without space.
-
-```js
-var foo = bar|0; // `foo` is forced to be signed 32 bit integer
-```
+requires spaces around all operators except `**`,
+does not allow spaces around `**`,
+and doesn't care whether spaces are used around `*`.
 
-The following patterns are considered problems:
+Without options to override the default settings,
+the following patterns are considered problems:
```

Comment:
On the ESLint website, we have color-coded backgrounds for code examples depending on if they're meant to show problems or correct code. Those backgrounds are generated based on the presence of certain wordings.

Basically, for problem sections, one of these prefixes should be used:

* `Examples of **incorrect** code`
* `Example of **incorrect** code`
* `Examples of additional **incorrect** code`
* `Example of addition **incorrect** code`

And you can put whatever you like after them. So for these lines, I would suggest something like: `Examples of **incorrect** code with the default settings:`